types.go

```diff
@@ -499,4 +499,13 @@ type PendingFee struct {
 	FeeTx             *wire.MsgTx
 }
 
+type VSPInfo struct {
+	Host string
+	Info *VspInfoResponse
+}
+
+type VSPList struct {
+	List []*VSPInfo
+}
```

Comment:
This would not work as expected as maps do not support indexing, and i would need to convert back to a slice in the target application (godcr).
Also, `Host` is not a property in `*VspInfoResponse` hence having to get the key `Host` for use later on is a challenge.